prefetch.bs

```diff
@@ -327,6 +333,12 @@ A <dfn export>cross-origin prefetch IP anonymization policy</dfn> has an <dfn ex
 
 <h2 id="html-patches">HTML Patches</h2>
 
+This section contains patches to [[HTML]].
+
+Add an additional item to [=navigation params=] as follows:
+:  <dfn for="navigation params">delivery type</dfn>
+:: a [=string=] (corresponding to {{PerformanceResourceTiming}} [=PerformanceResourceTiming/delivery type=]), default ""
+
 <div algorithm>
     <div class="note">This is extracted from <a spec=HTML>create navigation params by fetching</a>.</div>
```

Comment:
This note now looks like it's applying to the navigation params patch. Maybe add an `<hr>` before the `<div algorithm>`.

Reply:
Done.